ci/packaging-pipeline.yml

```diff
@@ -192,4 +197,4 @@ jobs:
 
   - publish: $(ARTIFACT_DIRECTORY)/demo_data-$(buildId).sql
     artifact: Demo SQL Data
-    displayName: Publish Demo SQL Data
+    displayName: Publish Demo SQL Data
```

Comment:
nit: Let's make sure that all files have a trailing newline. This is usually just an IDE setting you can turn on so it happens automatically when you save a file.

Reply:
Fixed